### DISCUSSION TITLE
LLM models paths

### DISCUSSION THREAD

**Discussion by magimyster** [comfyanonymous/ComfyUI#5591] [Q&A]
Why aren't the models in the LLM folder loaded from external paths?
They are very heavy models... like the one from Omnigen... if anyone has managed to move them from the original ComfyUI folder to a folder on an external hard drive, could they explain how they did it? For all the other models, I used the extra_model_paths.yaml file... but it seems that for the models in the LLM folder, they are not being considered.

**Comment by ltdrdata** [comment]
LLM model folder isn't belongs to Comfy Core.

If you want to specify the LLM model folder in extra_model_paths.yaml, request this feature from the custom node repository that uses the LLM model folder.

**Reply by magimyster** [reply]
Thank you

**Comment by phazei** [comment]
LLM's are becoming more popular and more integrated into ComfyUI flows.  Maybe it should be mapped and part of core now?

**Comment by sebsken** [comment]
There is a simple way to share your models folder or files across multiple places on windows: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/mklink
This is manual but it should work. It works for folders.
On linux, symbolic links are easy to do https://www.geeksforgeeks.org/how-to-symlink-a-file-in-linux/
